src/e2e-test/features/bigquery/source/BigQueryToBigQuery.feature

```diff
@@ -262,3 +262,246 @@ Feature: BigQuery source - Verification of BigQuery to BigQuery successful data
     Then Open and capture logs
     Then Verify the pipeline status is "Succeeded"
     Then Validate the values of records transferred to BQ sink is equal to the values from source BigQuery table
+
+
+  @BQ_UPSERT_SOURCE_TEST @BQ_UPSERT_SINK_TEST
+  Scenario: Verify scenario form BigQuery To to ensure that upsert operations are performed without updating the destination table
+    Given Open Datafusion Project to configure pipeline
+    When Expand Plugin group in the LHS plugins list: "Source"
+    When Select plugin: "BigQuery" from the plugins list as: "Source"
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "BigQuery" from the plugins list as: "Sink"
+    Then Connect plugins: "BigQuery" and "BigQuery2" to establish connection
+    Then Navigate to the properties page of plugin: "BigQuery"
+    And Enter input plugin property: "referenceName" with value: "Reference"
```

Comment:
We have to use the use connection step in all the scenarios as per ITN class. For more info we can connect on this